### PR TITLE
fix for issue #18

### PR DESCRIPTION
**Pull request by matthewkeil**

### CODE REVIEW

File: src/useMetamask.js [hunk 1]
```diff
@@ -18,6 +18,13 @@ const chains = (chainId) => {
 
 const useMetamask = () => {
   const state            = useContext(MetaStateContext);
+  const state = useContext(MetaStateContext);
```
Comment (mdtanrikulu) [inline src/useMetamask.js:21]: duplicate with above, you can remove this

**Comment by mdtanrikulu** [comment]
Heya! Thank you for your contribution! May I ask you to remove package-lock.json and dist package from the PR?

**Comment by GAlcaraz** [comment]
Hey! Wondering what happened to this PR, as it seems the fix is ready. Seems a shame it still can't be used with NextJS without the workaround.

**Review by mdtanrikulu — APPROVED** [review]

**Comment by mdtanrikulu** [comment]
I was away from maintaining the package for some time, was waiting some changes from the pr owner as well, but applying it now.

**Pull request merged**